scripts/undemultiplexed_index.py

```diff
@@ -171,8 +173,10 @@ def _QC(self, target_file, sample_info):
         QC2 = (Q30 >= self.QC_thresholds['%Q30'])
         QC3 = (self.read_pairs >= self.QC_thresholds['nr_read'])
         if QC1 and QC2 and QC3:
```

Comment:
could you add

``` and 'Include reads' in taget_file.udf ```

? This should ensure that the script does not fail if the udf field is not set